use-cases/using-vsocks.md

```diff
@@ -0,0 +1,133 @@
+# Kata Containers and vsocks
+
+- [Introduction](#introduction)
+    - [proxy communication diagram](#proxy-communication-diagram)
+    - [vsock communication diagram](#vsock-communication-diagram)
+- [System requirements](#system-requirements)
+- [Advantages of using vsocks](#advantages-of-using-vsocks)
+    - [High density](#high-density)
+    - [Reliability](#reliability)
+
+## Introduction
+
+There are two different ways to communicate processes in the virtual machine with
+processes in the host. The first one is by using serial ports, the processes in
+the virtual machine can read/write data from/to a serial port device and the
+processes in the host can read/write data from/to a unix socket. This solution
+has been part of QEMU from long time ago and almost all GNU/Linux distributions
+have support for serial ports, making it the most portable solution. However,
+there is a limitation that impact in its usage and implementation, the
```

Comment:
s/impact in its/impacts its/

Reply:
I'd also probably also swap the `,` after `implementation` to a `-` or `.`, to flow the sentence better.